plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-variations.php

```diff
@@ -29,7 +29,7 @@
 				require __DIR__ . '/html-product-attribute-inner.php';
 			?>
 				<div class="toolbar">
-					<button type="button" class="button button-primary save-variations"><?php esc_html_e( 'Save variations', 'woocommerce' ); ?></button>
+					<button type="button" class="button button-primary create-variations"><?php esc_html_e( 'Create variations', 'woocommerce' ); ?></button>
```

Comment:
This change appears to make sense based on my understanding of how the UI is supposed to function, but I don't see this change requirement documented anywhere, so I'm wondering if it is correct.